### PR TITLE
add instantiated systemd unit

### PR DESCRIPTION
**Pull request by satan1st**

### CODE REVIEW

File: scripts/clatd@.systemd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#
+# clatd service file for systemd
+#
+# Install it to:  /etc/systemd/system/clatd.service
+# Enable it with: systemctl enable clatd.service
+# Start it with:  systemctl start clatd.service
```
Comment (toreanderson) [inline scripts/clatd@.systemd:6]: Are you certain that this is correct? Also, shouldn't `Makefile` install it to where it needs to go?

File: scripts/clatd@.systemd [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# 
+# Written by Tore Anderson <tore@fud.no>
```
Comment (toreanderson) [inline scripts/clatd@.systemd:8]: Not entirely correct...

File: scripts/clatd@.systemd [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+[Unit]
+Description=464XLAT CLAT daemon
+Documentation=man:clatd(8)
+After=network-online.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/clatd -c /etc/clatd/%i plat-dev=%i
```
Comment (toreanderson) [inline scripts/clatd@.systemd:19]: Maybe this should have been `/etc/clatd/%i.conf` instead?

File: scripts/clatd@.systemd (hunk 2 above)
Comment (satan1st) [reply scripts/clatd@.systemd:8]: Going to fix

File: scripts/clatd@.systemd (hunk 3 above)
Comment (satan1st) [reply scripts/clatd@.systemd]: done

**Pull request merged**